### ISSUE TITLE
Show the time interval in addition to the absolute time in the bottom left-hand corner text

### ISSUE THREAD

**Issue by carlcotner** [nriley/Pester#25]
When selecting and entering "in 150 minutes", for example, the bottom left-hand corner text shows what the time will be in 150 minutes, which might be (for example)

> Alarm will be set for 1:00:00 AM tomorrow,
> Saturday, June 2, 2018.

This is, obviously, very useful.

When selecting and entering "at 1:00 AM June 2, 2018" (for example), the bottom left-hand corner text does not indicate that the time interval will be 150 minutes. At least for me it would be helpful if it included the time interval so I could double check whether the alarm was set correctly.

**Comment by nriley** [comment]
I agree and this feature already exists, but it is broken on the current macOS.

![image](https://user-images.githubusercontent.com/68303/40874949-59594694-6645-11e8-8b4a-dfdb55a4d5af.png)

The way this is intended to work for "at" alarms is that you'll get a confirmation of the time until your alarm which will pop up if you have Growl installed; if it doesn't look right, you can at least delete/re-set the alarm. If this is the next alarm, of course you can also look at the time displayed over the Dock icon to make sure it makes sense, too.

Unfortunately the old Growl framework I'm using broke at some point and I didn't notice that these confirmations stopped popping up. (I barely use Pester any more — I don't have a Mac at work and am mostly on iOS at home these days.)

So a solution, if you want the above, is to install Growl and use [Growl Version Detective](http://growl.info/downloads) to upgrade the Growl framework in Pester to the current version.

The real solution is that Pester should upgrade to a current Growl framework. The reason I didn't do so earlier is that the newer SDK was causing crashes, but I think these are fixed now.

**Comment by carlcotner** [comment]
Thanks for your reply and the info!
